docker-compose.yml

```diff
@@ -24,21 +24,34 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: TEST!timescale
       POSTGRES_DB: TEST
+      # Performance tuning parameters
+      max_wal_size: 8GB                     # Increased from 4GB to reduce checkpoint frequency
+      min_wal_size: 2GB                     # Increased from 1GB
+      shared_buffers: 1GB
+      effective_cache_size: 3GB
+      maintenance_work_mem: 256MB
+      checkpoint_completion_target: 0.9
+      checkpoint_timeout: '30min'           # Added to control checkpoint frequency
+      wal_buffers: '16MB'                  # Added for better WAL performance
+      random_page_cost: 1.1                # Added for better query planning
+      work_mem: '64MB'                     # Added for better query performance

```

Comment:
I think we can discard this part, but this helped to reduce the number of notifications about Wal.